analyses/interaction-plots/scripts/02-process_mutations.R

```diff
@@ -158,6 +160,12 @@ option_list <- list(
     default = 0,
     help = "Minimum sequencing depth for called mutations",
     metavar = "numeric"
+  ),
+  make_option(
+    opt_str = "--write_zenodo_csv",
+    action = "store_true",
+    default = FALSE,
+    help = "When this flag is used, output tabular data associated with manuscript figures targeted for Zenodo upload to file."
```

Comment:
if I were a stickler, I would want this to include a check that we were really using all data here, but I think it is fine.

Reply:
🎼